internal/daemonset/daemonset.go

```diff
@@ -310,3 +322,44 @@ func OverrideLabels(labels, overrides map[string]string) map[string]string {
 
 	return labels
 }
+
+func MakeLoadCommand(spec ootov1alpha1.ModprobeSpec) []string {
+	loadCommand := []string{"modprobe"}
+
+	if ra := spec.RawArgs; ra != nil && len(ra.Load) > 0 {
+		return append(loadCommand, ra.Load...)
+	}
+
+	if a := spec.Args; a != nil && len(a.Load) > 0 {
+		loadCommand = append(loadCommand, a.Load...)
+	} else {
+		loadCommand = append(loadCommand, "-v")
+	}
+
+	if dirName := spec.DirName; dirName != "" {
+		loadCommand = append(loadCommand, "-d", dirName)
+	}
+
+	loadCommand = append(loadCommand, spec.ModuleName)
+	return append(loadCommand, spec.Parameters...)
+}
+
+func MakeUnloadCommand(spec ootov1alpha1.ModprobeSpec) []string {
+	unloadCommand := []string{"modprobe"}
+
+	if ra := spec.RawArgs; ra != nil && len(ra.Load) > 0 {
+		return append(unloadCommand, ra.Load...)
+	}
```

Comment:
Is there a way to tell the CRD to respect mutual exclusion between using `paramaters/args` and using `rawArgs`? if not it means that if a user set them both we will ignore `parameters/args`.

Reply:
I think that is okay if it is documented in the CRD. Do you agree?

Reply:
If there is a possibility to enforce it by kubebuilder or something I think it is better, if not, let's just document it.

Reply:
I don't think OpenAPI constraints are that flexible. But this check could be made by the [validating webhook](https://issues.redhat.com/browse/MGMT-10520) we are working on that verifies the regexes in the mappings.